configuration/configuration-mainnet.yaml

```diff
@@ -177,3 +177,9 @@ TraceTxOutbound: False
 
 # Trace TxSubmission protocol messages.
 TraceTxSubmissionProtocol: False
+
```

Comment:
Please propagate `TraceHandshake` & `TraceLocalHandshake` to all the configuration yaml files